app/src/main/java/org/sopt/carrot/domain/repository/SearchRepository.kt

```diff
@@ -0,0 +1,7 @@
+package org.sopt.carrot.domain.repository
+
+import org.sopt.carrot.data.model.SearchProductModel
+
+interface SearchRepository {
+    suspend fun getSearchResults(keyword: String?): Result<Pair<List<SearchProductModel>, List<SearchProductModel>>>
```

Comment:
p1: 연쇄 효과인데,,ㅋㅋ 이 부분도 분리작업 반영하면 이런식으로 될거 같습니다!
```suggestion
    suspend fun getSearchResults(keyword: String?): Result<SearchProduct>
```